src/glue/jobs/s3_to_json.py

```diff
@@ -60,6 +61,24 @@ def write_file_to_json_dataset(
                 # Load JSON string as dict
                 j["Value"] = json.loads(j["Value"])
             data.append(j)
+            if dataset_identifier == "EnrolledParticipants":
+                if "Symptoms" in j["CustomFields"]:
+                    if len(j["CustomFields"]["Symptoms"]) > 0:
```

Comment:
Nit: This can be rolled up into one if statement:

```
if "Symptoms" in j["CustomFields"] and len(j["CustomFields"]["Symptoms"]) > 0:
```

Reply:
It could be, but then I would need to add another `if "Symptoms" in j["CustomFields"]` statement on line 80 and that would be redundant.

Reply:
Ah, that makes sense - I should've been more thorough.